chia/cmds/wallet_funcs.py

```diff
@@ -162,10 +162,16 @@ async def add_token(args: dict, wallet_client: WalletRpcClient, fingerprint: int
     token_name = args["token_name"]
```

Comment:
Maybe consider updating the name to help indicate to readers that this adds or renames a wallet.

Reply:
I was pondering this, but do you have any ideas?  `manage_token` is the best I can come up with but that sounds far too sophisticated for what this does.

Reply:
add_or_rename_token? or is that too wordy? manage_token would be ok too I think since manage doesn't have a single meaning.

Reply:
I just feel like if it's manage_token, people will wonder where to add a token.  And the functionality is so small and similar I don't think it's worth splitting into two commands.  Renaming can be thought of as overwriting one of your current tokens with an add :)